### DISCUSSION TITLE
Get last 10-50 posts from user (HELP PLEASE)

### DISCUSSION THREAD

**Discussion by ummataliyev** [kevinzg/facebook-scraper#1077] [Q&A]
from facebook_scraper import get_posts
from pprint import pprint
from itertools import islice

posts = get_posts(
    post_urls=["https://www.facebook.com/ibrat.jumaboyev"],
    credentials=("sapienscl.com", ""),
)

for post in islice(posts, 5):
    pprint(post['text'])

how can i get users last n posts with facebook-parcer I can not do it. Please help me)

**Comment by evildrome** [comment]
There's no specific built in way to filter by poster.

You'd need to put code around the scraper to count how many posts it gets from the user you are interested in and stop scraping when you reach the limit you set.
